docs/src/options/path_prefix.md

```diff
@@ -0,0 +1,11 @@
+# Path Prefix
+
+Sometimes your website may not be hosted at the root at of a domain and your nav links will not be point to the right place.
```

Comment:
this is either missing a word or has an extra one :)

Reply:
how did I write a book ashley wat??